Packages/vcs/vcs/vector.py

```diff
@@ -190,11 +205,11 @@ class Gv(object):
     vc.xyscale('linear', 'area_wt')     # Will set them both
 
     Specify the line style:
-     vc.line=0              # Same as vc.line='solid'
-     vc.line=1              # Same as vc.line='dash'
-     vc.line=2              # Same as vc.line='dot'
-     vc.line=3              # Same as vc.line='dash-dot'
-     vc.line=4              # Same as vc.line='long-dot'
+     vc.linetype=0              # Same as vc.linetype='solid'
```

Comment:
@doutriaux1 This is a comment in the sources, the user should use strings instead of numbers indeed.